man/systemd.network.xml

```diff
@@ -2405,6 +2405,16 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
         network traffic.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>UplinkInterface=</varname></term>
+        <listitem><para>Specifies name or index of uplink interface, or the special value
+        <literal>%none</literal>. When emitting DNS, NTP, or SIP servers are enabled but no servers are
+        specified, the servers configured in the uplink interface will be emitted. When unset, the link
+        which has default gateway with higher priority will be automatically selected. When
+        <literal>%none</literal>, no uplink interface will be selected. Defaults to unset.
```

Comment:
have we used a special value like this elswehere so far? i don't think we did? 

(not against this, i have no better idea of a special string that definitely is not a compatible ifname could be)

Reply:
or maybe just a `-`?